<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticsTrue</class>
 <widget class="QWidget" name="StatisticsTrue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>549</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>479</width>
    <height>549</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>479</width>
    <height>549</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QPushButton" name="exit_but">
     <property name="text">
      <string>Выйти</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="show_statisticsfalse_but">
     <property name="text">
      <string>Показать ошибочные вводы</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="save_result_but">
     <property name="text">
      <string>Сохранить результат</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QTextBrowser" name="textBrowser">
     <property name="minimumSize">
      <size>
       <width>457</width>
       <height>192</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>457</width>
       <height>192</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>455</width>
        <height>290</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
